pyspark_datasources/jsonplaceholder.py

```
@@ -0,0 +1,222 @@
+                    "address_geo_lng STRING, company_name STRING, company_catchPhrase STRING, "
+                    "company_bs STRING")
+        elif endpoint == "todos":
+            return "userId INT, id INT, title STRING, completed BOOLEAN"
+        elif endpoint == "comments":
+            return "postId INT, id INT, name STRING, email STRING, body STRING"
+        elif endpoint == "albums":
+            return "userId INT, id INT, title STRING"
+        elif endpoint == "photos":
+            return "albumId INT, id INT, title STRING, url STRING, thumbnailUrl STRING"
+        else:
+            return "userId INT, id INT, title STRING, body STRING"
```

Ruff (0.12.2)

Use a dictionary instead of consecutive `if` statements

(SIM116)